src/pages/Productlist.js

```diff
@@ -1,8 +1,75 @@
-import React from 'react'
+import {useState, useEffect} from 'react'
+import '../style/productlist.css'
+import Product from '../components/Product'
+import { useFilter } from '../context/filter-context'
+import Filter from '../components/Filter'
+import getSortedProduct from '../utils/sorting';
+import filteredByRating from '../utils/rating';
+import priceSorting from '../utils/price'
+import searchQuery from '../utils/searchQuery';
+import {useParams} from 'react-router-dom'
```

Comment:
you can do barrel import here
[check here](https://dev.to/justmyrealname/organize-react-components-better-with-barrel-exports-4fno)